examples/tutorials/ItWaspsOnMyMachine/migrations/20220819110846_init/migration.sql

```diff
@@ -1,5 +0,0 @@
--- CreateTable
-CREATE TABLE "Excuse" (
```

Comment:
This file was causing DB errors. We have two init migrations, this was the one that was out of date. I kept the other one.